dlrs/main/classes/ManageLookupRollupSummariesController.cls

```diff
@@ -156,6 +174,50 @@ public with sharing class ManageLookupRollupSummariesController {
     return null;
   }
 
+  public void hasChildTriggers() {
+    if (LookupRollupSummary == null) {
+      return;
+    } else {
+      try {
+        RollupSummary rs = new RollupSummary(LookupRollupSummary);
+        String childTrigger = RollupSummaries.makeTriggerName(rs);
+        String parentTrigger = RollupSummaries.makeParentTriggerName(rs);
+        ApexTriggersSelector selector = new ApexTriggersSelector();
+        Map<String, ApexTrigger> loadTriggers = selector.selectByName(
+          new Set<String>{ ChildTrigger, ParentTrigger }
+        );
+
+        switch on loadTriggers.size() {
+          when 2 {
+            Apexpages.addMessage(
+              new Apexpages.Message(
+                Apexpages.severity.CONFIRM,
+                'Rollup has child and parent triggers deployed. Can activate'
+              )
+            );
+          }
+          when 1 {
+            Apexpages.addMessage(
+              new Apexpages.Message(
+                Apexpages.severity.WARNING,
+                'Rollup has only one trigger deployed. Cannot activate'
+              )
+            );
+          }
+          when else {
+            Apexpages.addMessage(
+              new Apexpages.Message(
+                Apexpages.severity.FATAL,
+                'Rollup does not have any triggers deployed. Cannot activate'
+              )
+            );
+          }
+        }
+      } catch (Exception e) {
+      }
```

Comment:
please add a comment that indicates exception has been left empty intentionally + reasons.